ui/pages/send/gas-display/gas-display.js

```diff
@@ -267,7 +267,10 @@ export default function GasDisplay({ gasError }) {
                 />
               }
             />,
-            (gasError || isInsufficientTokenError) && (
+            (gasError ||
+              isInsufficientTokenError ||
+              draftTransaction.asset.balance === '0x0' ||
+              draftTransaction.asset.balance === '0x00') && (
```

Comment:
in what cases is it '0x00' ?

Reply:
I am not sure, but sometimes it is '0x0' and sometimes '0x00' i dont know why